tasty-golden-extra.cabal

```
@@ -31,17 +31,17 @@ source-repository head
 
 common deps
   build-depends:
-    , aeson           ==2.1.2.1    || ==2.2.3.0
+    , aeson           >=2.1.2.1    && <2.2.3.0.0
     , aeson-diff      ^>=1.1.0.13
     , aeson-pretty    ^>=0.8.10
-    , base            ==4.14.3.0   || ==4.15.1.0  || ==4.16.3.0 || ==4.17.1.0 || ==4.18.2.1 || ==4.19.1.0
-    , bytestring      ==0.10.12.0  || ==0.10.12.1 || ==0.11.3.1 || ==0.11.4.0 || ==0.11.5.3 || ==0.12.1.0
+    , base            >=4.14.3.0   && <4.19.1.0.0
+    , bytestring      >=0.10.12.0  && <0.12.1.0.0
     , pretty-show     ^>=1.10
     , pretty-simple   ^>=4.1.3.0
     , tasty           ^>=1.5.2
     , tasty-discover  ^>=5.0.0
     , tasty-golden    ^>=2.3.5
-    , text            ==1.2.4.1    || ==1.2.5.0   || ==2.0.2    || ==2.1.1
+    , text            >=1.2.4.1    && <2.1.1.0
```

```suggestion
    , text            ^>=1.2.4.1    || >=2.0 && <2.2
```
Because if there is ever a `text-1.3`, it will be allowed to break relative to `text-1.2.4.1`.